e2e-tests/adapters/src/pages/routes/ssr/remote-file.jsx

```diff
@@ -109,3 +103,11 @@ export const pageQuery = graphql`
 `
 
 export default RemoteFile
+
+export function getServerData() {
+  return {
+    props: {
+      ssr: true,
+    },
+  }
+}
```

Comment:
This page wasn't actually SSR :yikes: Making it SSR uncovered some other problems that this PR is also fixing